src/Authenticator/LoginHandler.php

```diff
@@ -77,11 +77,13 @@ public function doLogin($data, MemberLoginForm $form, HTTPRequest $request)
     /**
      * Action handler for loading the MFA authentication React app
      *
-     * @return array|HTTPResponse
+     * @return array template variables {@see SilverStripe\Security\Security::renderWrappedController}
      */
     public function mfa()
     {
-        return [];
+        return [
```

Comment:
Considering this was really confusing for us can we mention that the template for these template variables is `Page.ss` somewhere?

Reply:
Doesn't need to be Page.ss though... that's (part of) the issue.

Reply:
Could probably mention that it's "normally" `Page.ss` though :)